src/atomate2/forcefields/utils.py

```diff
@@ -59,9 +61,18 @@ def ase_calculator(calculator_meta: str | dict, **kwargs: Any) -> Calculator | N
             calculator = PESCalculator(potential, **kwargs)
 
         elif calculator_name == MLFF.MACE:
-            from mace.calculators import mace_mp
-
-            calculator = mace_mp(**kwargs)
+            import torch
+            from mace.calculators import MACECalculator, mace_mp
+
+            model = kwargs.get("model")
+            if isinstance(model, str | Path) and os.path.isfile(model):
+                model_path = model
+                calculator = MACECalculator(
```

Comment:
Think we need to change this to:
```
device = kwargs.get("device") or "cpu"
calculator =  MACECalculator(
    model_paths=model_path,
    device = device,
    **kwargs
)
```

Reply:
Thanks. One needs to remove the "device" key from the kwargs. Otherwise, it will complain that it is set twice. Will do that.